### PR TITLE
224: Ellipsis hint to overflowing text

### PR DESCRIPTION
**Pull request by srameshr**

### CODE REVIEW

File: src/components/addressbook/form.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ const Render = ({ handleSubmit, blockAddress, invalid, pristine, submitting, can
                         component={renderTextField}
                         type="text"
                         label="Address Name"
-                        validate={ required } />
+                        disabled />
```
Comment (elaineo) [inline src/components/addressbook/form.js]: why is this field disabled?

Comment (srameshr) [inline src/components/addressbook/form.js]: In this [bug](https://github.com/ethereumproject/emerald-wallet/issues/226) he mentions *"Address name should not be a required field. _After adding an address name you cannot remove it again._ You should not be forced to have an address name."*

So updates are not possible to the address field.

Comment (elaineo) [inline src/components/addressbook/form.js]: ok, we can make the field optional without disabling it altogether.

Comment (srameshr) [inline src/components/addressbook/form.js]: Sure, Il just update it.

**Comment by srameshr** [comment]
@elaineo Done.

File: src/components/addressbook/form.js (hunk 1 above)
Comment (srameshr) [inline src/components/addressbook/form.js]: @elaineo Done

**Pull request closed without merging**